mongodb/activity/queryDocument/README.md

```diff
@@ -0,0 +1,82 @@
+<!--
+title: MongoDB Query Document
+weight: 4622
+-->
+# MongoDB Query Document
+This activity allows you to query for one or more documents from a collection in MongoDb database based on a matching criteria entered as a JSON
+
+## Installation
+
+### Flogo CLI
+```bash
+flogo install github.com/project-flogo/datastore-contrib/mongodb/activity/queryDocument
+```
+
+## Configuration
+
+### Settings:
+| Name                   | Type       | Description
+| :---                   | :---       | :---    
+| mongoConnection        | connection | Choose a MongoDB connection from the drop down  - ***REQUIRED***
```

Comment:
Why not call it just "connection", isn't the mongodb redundant?

Reply:
Done